src/app/replayer/replayer.ml

```diff
@@ -604,7 +604,7 @@ let write_replayer_checkpoint ~logger ~ledger ~last_global_slot_since_genesis
       let%map input =
         create_replayer_checkpoint ~ledger ~start_slot_since_genesis
       in
-      input_to_yojson input |> Yojson.Safe.pretty_to_string
+      input_to_yojson input |> Yojson.Safe.to_string
```

Comment:
How large is the json output? Do we need it at all, now it is non-pretty, but maybe it's worth dropping right away?